docs/cloud/earthly-ci.md

```diff
@@ -0,0 +1,149 @@
+
+You can open the Earthly Web UI by running:
+
+```bash
+earthly web
+```
+
+This will associate your Earthly account with your GitHub login and then take you to the Earthly Web UI.
+
+### 5. Create your first Earthly pipeline
+
+In the Earthly Web UI, under the Organizations drop-down in the top-left of the screen, select the organization that was granted access to Earthly CI. Then follow the instructions on the screen to create a new Earthly CI project, add your first repository, create your pipeline via a new or an existing Earthfile, and then run your first build.
+
+For your very first CI pipeline you can use the following example:
+
+```Earthfile
+VERSION 0.7
+PROJECT my-org/my-project
+
+FROM alpine:3.15
+
+my-pipeline:
+  PIPELINE
+  TRIGGER push main
+  TRIGGER pr main
+  BUILD +my-build
+
+my-build:
+  RUN echo Hello world
+```
+
+This example shows a simple pipeline called `my-pipeline`, which is triggered on either a push to the `main` branch, or a pull request against the `main` branch. The pipeline executes the target `my-build`, which simply prints `Hello world`.
+
+Pipelines and their definitions, including their triggers must be merged into the primary branch (which, unless overridden, is the default branch on GitHub -- usually `main`) in order for the triggers to take effect.
+
+Please note that the `PROJECT` declaration needs to match the name of your organization and the name of your Earthly project that you create in the Earthly Web UI.
+
+### 6. Set up registry access
+
+During your experimentation with Earthly CI, you may encounter DockerHub rate limiting errors. To avoid this, you can setup your DockerHub account by using the command `earthly registry setup`.
+
+```bash
+earthly registry --org <org-name> --project <project-name> \
+setup --username <registry-user-name> --password-stdin \
+<host>
+```
+
+If the registry is DockerHub, then you can leave out the registry host argument.
+
+You may additionally log into other registries, such as AWS ECR, or GCP Artifact Registry, by using the following:
+
+```bash
+# For AWS ECR
+earthly registry --org <org-name> --project <project-name> setup --cred-helper ecr-login --aws-access-key-id <key> --aws-secret-access-key <secret> <host>
+# For GCP Artifact Registry
+earthly registry --org <org-name> --project <project-name> setup --cred-helper gcloud --gcp-service-account-key <key> <host>
+```
+
+### 7. Invite your team
+
+A final optional step is to invite your team to use Earthly CI. This can be done by running:
+
+```bash
+earthly --org <org-name> org invite <email>
+```
+
+Or by using the Earthly Web UI.
+
+## Known limitations
+
+Please note that we are aware of the following ongoing issues:
+
+* The logs of certain builds do not show up in the UI sometimes. If you run into this problem, please let us know and provide us with the URL of the page!
+* Creating an account, or logging in directly in the web UI is not yet available. Use the `earthly web` command instead.
```

Comment:
Once a user has logged in with `earthly web` once they won't have to do it again.

Reply:
Right, but also there's no link on our website to take you to the login page. We could just add the link... but the signup experience is not there yet, so it's better not to add it. I would just leave this remark as is for now, for these reasons.